terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -161,7 +161,22 @@
 )
 
 other_public_keys = {
-    'dev': [],
+    'dev': [
+        (
```

Comment:
```suggestion
```

Parenthesis are not needed for the string concatenation since it is already in a square-bracketed block.

```
>>> ['abc'
...  'def']
['abcdef']
```

Reply:
True, however, it looks better parenthesized when there are more than one entries.